pkgs/development/python-modules/aiopvpc/default.nix

```diff
@@ -0,0 +1,66 @@
+{ lib
+, aiohttp
+, async-timeout
+, buildPythonPackage
+, fetchFromGitHub
+, fetchpatch
+, poetry-core
+, pytest-asyncio
+, pytest-timeout
+, pytestCheckHook
+, pythonOlder
+, pytz
+}:
+
+buildPythonPackage rec {
+  pname = "aiopvpc";
+  version = "2.0.2";
+  disabled = pythonOlder "3.7";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "azogue";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1ajs4kbdlfn4h7f3d6lwkp4yl1rl7zyvj997nhsz93jjwxbajkpv";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = [
+    aiohttp
+    pytz
+    async-timeout
+  ];
+
+  checkInputs = [
+    pytest-asyncio
+    pytest-timeout
+    pytestCheckHook
+  ];
```

Comment:
```
=============================== warnings summary ===============================
tests/test_pvpc.py: 11 warnings
  /nix/store/69jp5jhv1v79zk2dlv5vy9ssz89bik9w-python3.8-pytest-6.2.2/lib/python3.8/site-packages/_pytest/python.py:172: PytestUnhandledCoroutineWarning: async def functions are not natively supported and have been skipped.
  You need to install a suitable plugin for your async framework, for example:
    - anyio
    - pytest-asyncio
    - pytest-tornasync
    - pytest-trio
    - pytest-twisted
    warnings.warn(PytestUnhandledCoroutineWarning(msg.format(nodeid)))

-- Docs: https://docs.pytest.org/en/stable/warnings.html
================== 1 passed, 14 skipped, 11 warnings in 0.47s ==================
```

That's a bit disappointing. Can you investigate, why it skips so many tests even with `pytest-asyncio`?

Reply:
Some tests require network access. They are skipped. Let me see if we can increase the number of tests running successfully as there are fixtures present in the repo.

Reply:
Anything left to do here before merging?

Reply:
Hi there @fabaff and @lukegb, 
the testing environment in `aiopvpc` is using `pytest-aiohttp`, not `pytest-asyncio`. I could try to make the switch, if that's the problem?

Reply:
Just checked that I can change `pytest-aiohttp` for `pytest-asyncio`, add the decorators over async tests (`@pytest.mark.asyncio`), and everything runs as expected, so, no need to change anything outside `aiopvpc`, I'll publish another release with the change :)

Reply:
Thanks for the feedback, @azogue.